cmake/FindDPCPP.cmake

```
@@ -45,6 +45,11 @@ if(NOT "${DPCPP_SYCL_TARGET}" STREQUAL "")
   list(APPEND DPCPP_FLAGS "-fsycl-targets=${DPCPP_SYCL_TARGET};")
 endif()
 
+set(DPCPP_DISABLE_ITT_FOR_CUTLASS OFF CACHE BOOL "Disables linking of the Instrumentation and Tracing Technology (ITT) device libraries for VTune")
```

Previously you had the flag on by default

NIT: just my opinion that this is a bit nicer to read
```suggestion
option(DPCPP_DISABLE_ITT_FOR_CUTLASS "Disables linking of the Instrumentation and Tracing Technology (ITT) device libraries for VTune" ON)
```